service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -1396,15 +1396,17 @@ func (t *transferQueueActiveTaskExecutor) applyParentClosePolicy(
 		return nil
 
 	case enumspb.PARENT_CLOSE_POLICY_TERMINATE:
-		childNamespaceId, err := t.registry.GetNamespaceID(namespace.Name(childInfo.GetNamespace()))
+		childNamespaceID, err := t.registry.GetNamespaceID(namespace.Name(childInfo.GetNamespace()))
 		switch err.(type) {
-		case nil, *serviceerror.NotFound:
+		case nil:
+		case *serviceerror.NotFound:
```

Comment:
Fix the request cancel case below as well?

Reply:
This is :facepalm:^2.

Reply:
Updated unit test also.